### PR TITLE
update(release-notes): add 7.8.4 bug fixes

### PR DESCRIPTION
**Pull request by NathalieC**
* Add bug fixes made in 7.8.4
* Add UID-8 in 7.8.3

### CODE REVIEW

File: md/release-notes.md [hunk 1]
```diff
@@ -239,15 +239,45 @@ If you need to use custom types, it is recommended to use Business Object from t
 * BS-19105	Studio Import: unable to recognize git repositories after studio migration
 * BS-19158	Default SAML configuration doesn't work with AD FS
 
+#### Fixes in Bonita 7.8.4 (2019-04-11)
+#### Fixes in Engine component
+* BR-68	BDM REST API response JSON is huge
+* BR-69	SQLServerException error when deleting cases with more then 2100 subprocesses
+* BS-19024	Custom page update sometimes doesn't work unless you restart the platform
+* BS-19284	Cases open and archived tabs show inconsistent Display and Technical Process Name in the process Name column
+* BS-19295	SELECT widget sets the field of the variable holding the initial value to null when the page/form loads if it is shared by another SELECT widget
```
Comment (educhastenier) [inline md/release-notes.md]: some of those bugfixes are not **engine** fixes

File: md/release-notes.md [hunk 2]
```diff
@@ -239,15 +239,45 @@ If you need to use custom types, it is recommended to use Business Object from t
 * BS-19105	Studio Import: unable to recognize git repositories after studio migration
 * BS-19158	Default SAML configuration doesn't work with AD FS
 
+#### Fixes in Bonita 7.8.4 (2019-04-11)
+#### Fixes in Engine component
+* BR-68	BDM REST API response JSON is huge
+* BR-69	SQLServerException error when deleting cases with more then 2100 subprocesses
+* BS-19024	Custom page update sometimes doesn't work unless you restart the platform
+* BS-19284	Cases open and archived tabs show inconsistent Display and Technical Process Name in the process Name column
```
Comment (educhastenier) [inline md/release-notes.md]: This bugfixe is not an engine fix

**Review by educhastenier — CHANGES_REQUESTED** [review]

File: md/release-notes.md (hunk 2 above)
Comment (NathalieC) [reply md/release-notes.md]: Thank you Manu. Moving it to web

File: md/release-notes.md (hunk 1 above)
Comment (NathalieC) [reply md/release-notes.md]: It's a duplicate of UID-38. Removing it. Thanks.

File: md/release-notes.md [hunk 3]
```diff
@@ -239,15 +239,45 @@ If you need to use custom types, it is recommended to use Business Object from t
...
+
+#### Fixes in Studio component
+* BST-95 Revert change in staging view is not refreshing project view
+* BST-122 Studio crashes when the user tries to switch repository while the engine is still starting
+* BST-124 Organization - all memberships updated when adding/changing a role
+* BST-142 Importing 7.8.1 worskpace fails in 7.8.3
+* BST-177 Search index not built with la-builder
+* BST-210 Avoir Studio freeze if the user start to switch into a repository shared with git / svn just after the Studio launch
+* BST-221 UID allows to have 2 forms with same name when created from the Studio: this breaks LA Builder
+* BS-19331 Fail to import .bos file with only a Groovy script in Bonita Community Edition 7.8.3
```
Comment (alachambre) [inline md/release-notes.md]:
```suggestion
* BS-19331 Groovy scripts with a custom package are not exported correctly
```

File: md/release-notes.md [hunk 4]
```diff
@@ -239,15 +239,45 @@ If you need to use custom types, it is recommended to use Business Object from t
...
 
+#### Fixes in Bonita 7.8.4 (2019-04-11)
+#### Fixes in Engine component
+* BR-68	BDM REST API response JSON is huge
+* BR-69	SQLServerException error when deleting cases with more then 2100 subprocesses
+* BS-19024 Custom page update sometimes doesn't work unless you restart the platform
+* BS-19298 Process may end up locked forever
+
+#### Fixes in Studio component
+* BST-95 Revert change in staging view is not refreshing project view
```
Comment (alachambre) [inline md/release-notes.md]:
```suggestion
* BST-95 Revert change in the Git staging view is not refreshing project view
```

**Pull request merged**